content/en/blog/_posts/2021-04-22-gateway-api/2021-04-22-gateway-api.md

```diff
@@ -0,0 +1,199 @@
+
+---
+layout: blog
+title: 'Evolving Service networking with the Gateway API'
+date: 2021-04-22
+slug: gateway-api
```

Comment:
```suggestion
slug: evolving-service-networking-with-the-gateway-api
```
Small thing, but the slug should ideally match the title of the page^^;;;

Reply:
I have updated the title and the slug to "Evolving Kubernetes networking with the Gateway API"